### PR TITLE
PDA-60 Redesign Task Model and refactored TaskService to use Kie Client Library

### PR DESCRIPTION
**Pull request by ffleandro**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: src/main/java/org/entando/plugins/pda/pam/service/task/KieTaskCommentService.java [hunk 1]
```diff
@@ -32,10 +33,20 @@
...
-                .collect(Collectors.toList());
+        try {
+            return client.getTaskCommentsByTaskId(taskId.getContainerId(), taskId.getInstanceId())
+                    .stream()
+                    .map(KieTaskCommentService::dtoToComment)
+                    .collect(Collectors.toList());
+        } catch (KieServicesHttpException e) {
+            if (e.getHttpCode().equals(HttpStatus.NOT_FOUND.value())/*
+                    //Some endpoints return 500 instead of 404
+                    || e.getHttpCode().equals(HttpStatus.INTERNAL_SERVER_ERROR.value())*/) {
```
Comment (werbth) [inline src/main/java/org/entando/plugins/pda/pam/service/task/KieTaskCommentService.java]: Remove commented code. I think the whole comment can be removed.

**Review by werbth — CHANGES_REQUESTED** [review]

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Review by werbth — APPROVED** [review]

**Comment by werbth** [comment]
Retest this please

**Comment by entando-service** [comment]
All is well

**Pull request merged**